src/sentry/dynamic_sampling/tasks.py

```diff
@@ -103,6 +110,61 @@ def recalibrate_orgs() -> None:
             logger.info("Dynamic sampling organization recalibration failed", extra=errors)
 
 
+def rebalance_org(org_volume: OrganizationDataVolume) -> Optional[str]:
+    """
+    Calculates the rebalancing factor for an org
+
+    It takes the last interval total number of transactions and kept transactions, and
+    it figures out how far it is from the desired rate ( i.e. the blended rate)
+    """
+
+    redis_client = get_redis_client_for_ds()
+    factor_key = generate_cache_key_rebalance_factor(org_volume.org_id)
+    # we need a project from the current org... unfortunately get_blended_sample_rate
+    # takes a project (not an org)
+    # TODO RaduW is there a better way to get an org Project than filtering ?
+    org_projects = Project.objects.filter(organization__id=org_volume.org_id)
```

Comment:
`get_blended_sample_rate` can take either an org or a project, see the doc string on that method. Please verify with the implementation in getsentry, but based on the signature description, you can run:

```py
desired_sample_rate = quotas.get_blended_sample_rate(project=None, organization_id=org_volume.org_id)
```

Reply:
I have updated the signature of `get_blended_sample_rate` yesterday. I did not update it in the previous usages across Sentry. I will do it in a separate PR. The code you are seeing here has only been moved.